rrmngmnt/network.py

```diff
@@ -141,6 +142,11 @@ def __init__(self, host):
         super(Network, self).__init__(host)
         self._m = _session(host)
         self._hnh = None
+        self._nmcli = NMCLI(host)
```

Comment:
I would put this line instead of 149. or set it to None, and create instance only first time:

```python
    @property
    def nmcli(self):
        if self.nmcli is None:
            self._nmcli = NMCLI(self.host)
        return self._nmcli
```